var/spack/repos/builtin/packages/teuchos/package.py

```diff
@@ -19,98 +19,76 @@
 #
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
-
-import sys
+import subprocess
 
 from spack import *
 
-#def trilinosez_depends_on_test(spec, **kwargs):
-#    depends_on('mpi')
 
 class Teuchos(CMakePackage):
     """FIXME: Put a proper description of your package here."""
 
     # FIXME: Add a proper url for your package's homepage here.
-    homepage = "https://www.example.com"
-    git      = "ssh://git@github.com/MikolajZuzek/Trilinos.git"
-    url      = "http://localhost/0.1.0.tar.gz"
+    homepage = "http://localhost/"
+    url      = "http://localhost/teuchos-0.1.0.tar.gz"
 
     maintainers = ['keitat']
 
-    version('1-teuchos-cmake', branch='1-teuchos-cmake', preferred=True)
+    version('0.1.0', sha256=@@@@@, preferred=True)
```

Comment:
@marcinwrobel1986 Python can't parse this...

Reply:
@mzuzek yes, it is substituted with calculated sha256. I will try to use `file://` and maybe try to remove sha256. When still no success, then I will at least put @@@@@ inside quotes like '@@@@@', so then it will not raised an error.